src/main/java/org/kaazing/netx/ws/internal/io/IncomingSentinelExtension.java

```diff
@@ -22,6 +22,7 @@
 public final class IncomingSentinelExtension extends WebSocketExtensionSpi {
     private final WebSocketFrameConsumer cachedOnBinaryReceived;
     private final WebSocketFrameConsumer cachedOnContinuationReceived;
+    private final WebSocketFrameConsumer cachedOnCloseReceived;
     private final WebSocketFrameConsumer cachedOnPingReceived;
     private final WebSocketFrameConsumer cachedOnPongReceived;
     private final WebSocketFrameConsumer cachedOnTextReceived;
```

Comment:
Suggest naming these as `default...` instead of `cached...`.  What do you think?